javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+function countdown() {
+  var text = document.querySelector('h1');
+  if (text.textContent === '4') {
+    text.textContent = '3';
+  } else if (text.textContent === '3') {
+    text.textContent = '2';
+  } else if (text.textContent === '2') {
+    text.textContent = '1';
+  } else if (text.textContent === '1') {
+    text.textContent = '~Earth Beeeelooowww Us~';
+  }
```

Comment:
If you create a variable to decrement over time, I think you could do this without individual code blocks for each number